src/library_webgl.js

```diff
@@ -1307,6 +1307,17 @@ var LibraryGL = {
 #if MAX_WEBGL_VERSION >= 2
               case 0x8919: // GL_SAMPLER_BINDING
               case 0x8E25: // GL_TRANSFORM_FEEDBACK_BINDING
+              //buffer object binding
+              case 0x88ED: // GL_PIXEL_PACK_BUFFER_BINDING
+              case 0x88EF: // GL_PIXEL_UNPACK_BUFFER_BINDING
+              case 0x8C8F: // GL_TRANSFORM_FEEDBACK_BUFFER_BINDING
+              case 0x8A28: // GL_UNIFORM_BUFFER_BINDING
+              case 0x8F36: // COPY_READ_BUFFER_BINDING, same with COPY_READ_BUFFER
+              case 0x8F37: // COPY_WRITE_BUFFER_BINDING, same with COPY_WRITE_BUFFER
+
+              case 0x806A: // GL_TEXTURE_BINDING_3D
+              case 0x8C1D: // GL_TEXTURE_BINDING_2D_ARRAY
+              case 0x8CAA: // GL_READ_FRAMEBUFFER_BINDING
```

Comment:
Nit: code above this doesn't use the GL_ prefix, they should be the same (not sure which style the rest of the code uses)

Reply:
Yes. I should keep the style same.